app/views/api/applications/index.html.slim

```diff
@@ -1,5 +1,8 @@
 - service_name = @service.name
 - content_for :page_header_title, "Applications on #{service_name}"
+- content_for :page_header_body do
+  ' Applications are a representation of an API consumer. They include credentials obtained through the API's authentication method,
```

Comment:
> They include credentials obtained through the API's authentication method

I don't understand this. Should this be something along the lines of "creadentials to access the API"? Who and when obtained the credentials "through the API's authentication method"?